show/main.py

```diff
@@ -156,23 +150,46 @@ def summary(interfacename, sfp):
 
 # 'counters' subcommand
 @interfaces.command()
-def counters():
+@click.option('-p', '--period')
+@click.option('-c', '--clear', is_flag=True)
+def counters(period, clear):
```

Comment:
Shall we also provide -a as an argument option as -p and -c such that we don't have to print all the info by default?